src/test/java/com/google/sps/MatchRankingTest.java

```diff
@@ -331,4 +366,33 @@ private void testOnlyAddNewBio(String newBio, User user) {
     user.setBioMap(savedMatchWordCount);
   }
 
+  private String getRandomGoodBio(int bioLength) {
```

Comment:
Would it be useful to create a more generic function that can generate a bio using a mixture of words from different collections? This could allow you to create fake bios that have higher or lower keyword frequencies. 

I was thinking something like the following:
```
   private String generateRandomBio(int targetLength, float keywordPercentage, Collection<String> keywords, Collection<String> nonKeywords);
```

Reply:
This was a really good idea, I updated the function to take in keyword percentage so that I had more flexibility with the tests